ovis_robotiq_gripper/msg/OvisGripper_robot_output.msg

```diff
@@ -0,0 +1,6 @@
+uint8 rACT #requestActivation
```

Comment:
Same thing as my previous comment here

Reply:
Je suis d'accord avec ca!
Il y en a qui sont pas mal "to the point" exemple: "gPO = getPosition" mais il en ont qui sont plus dur a se rapeller même après avoir lu la datasheet et les commentaires que t'a laissé.

Reply:
It would be technically possible. It would also be a good deal of work, since these messages are the input type in their gripper object and the other used packages.
I do not think this change would currently be worth the time investment.

Reply:
@saxtot and I spoke face to face. And I must agree with keeping it this way. It will not be a node that's going to have a lot of work done. And if changing it doesn't really add much value, except for maintainability, I think we can keep it like this and move on since we have more important projects to do.